trajectory:
  # Common options to be used for all trajectories (these are the default values, so you can omit them in this case).
  frame_id: ""
  dt: 0.2
  delay: 0.0
  loop: false
  use_heading: true
  fly_now: false
  offset: [0.0, 0.0, 0.0, 0.0]      # [x, y, z, heading]

  # If 'dynamic_uav_name' is true, the 'uavs' parameter is ignored and trajectory is loaded
  # to the UAV, specified by the 'uav_name' parameter from the 'trajectory/filename' parameter.
  # This allows for UAV name-agnostic trajectory loading (it doesn't need to be specified in
  # the yaml and can be loaded eg. from an environment variable.
  dynamic_uav_name: false
  # Options specified here will override the common options above.
  # base_path: "trajectories"
  uavs:
    uav1:
      filename: "leader_1.0ms.txt"

service:
  timeout: 3     # maximal waiting time for response after calling service
  load_name: "control_manager/trajectory_reference"
  goto_name: "control_manager/goto_trajectory_start"
  track_name: "control_manager/start_trajectory_tracking"
  stop_name: "control_manager/stop_trajectory_tracking"
